api/src/helpers.py

```diff
@@ -58,39 +75,50 @@ def get_status(dag_run_id: str) -> Dict:
         f"https://{mwaa_cli_token['WebServerHostname']}/aws_mwaa/cli"
     )
 
-    raw_data = "dags list-runs -d veda_discover"
-    mwaa_response = requests.post(
+    raw_data_ingest = "dags list-runs -d veda_ingest"
+    mwaa_response_ingest = requests.post(
         mwaa_webserver_hostname,
         headers={
             "Authorization": "Bearer " + mwaa_cli_token["CliToken"],
             "Content-Type": "application/json",
         },
-        data=raw_data,
+        data=raw_data_ingest,
     )
-    decoded_response = base64.b64decode(mwaa_response.json()["stdout"]).decode("utf8")
-    rows = decoded_response.split("\n")
+    decoded_response_ingest = base64.b64decode(
+        mwaa_response_ingest.json()["stdout"]
+    ).decode("utf8")
+    rows_ingest = decoded_response_ingest.split("\n")
 
-    try:
-        matched_row = next(row for row in rows if dag_run_id in row)
-    except StopIteration:
-        raise Exception(f"Failed to find dag run id: {dag_run_id}")
+    raw_data_discover = "dags list-runs -d veda_discover"
```

Comment:
Sorry to be annoying. Does the state command help look up the specific run instead of all runs? You could also allow date as a parameter if you have concerns about runs going through midnight.
```suggestion
    raw_data_discover = f"dags state {dag_run_id} {datetime.now().isoformat()}"
```